service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandler.java

```diff
@@ -86,6 +96,28 @@ public void sendNotificationToGatekeeper(NotifyGatekeeperEvent event) {
         sendNotification(GATEKEEPER_SUBMISSION_TEMPLATE, email, parameters, reference);
     }
 
+    @EventListener
+    public void sendSDOToCafcass(SDOSubmittedEvent event) {
+        CaseDetails caseDetails = event.getCallbackRequest().getCaseDetails();
+        String localAuthorityCode = (String) caseDetails.getData().get(CASE_LOCAL_AUTHORITY_PROPERTY_NAME);
+        Map<String, Object> parameters = cafcassEmailContentProvider
+            .buildCafcassSDOSubmissionNotification(caseDetails, localAuthorityCode);
+        String reference = String.valueOf(caseDetails.getId());
+        String email = cafcassLookupConfiguration.getCafcass(localAuthorityCode).getEmail();
+        sendNotification(SDO_SUBMISSION_TEMPLATE, email, parameters, reference);
+    }
+
+    @EventListener
+    public void sendSDOToLocalAuthority(SDOSubmittedEvent event) {
+        CaseDetails caseDetails = event.getCallbackRequest().getCaseDetails();
+        String localAuthorityCode = (String) caseDetails.getData().get(CASE_LOCAL_AUTHORITY_PROPERTY_NAME);
+        Map<String, Object> parameters = localAuthorityEmailContentProvider
+            .buildLocalAuthoritySDOSubmissionNotification(caseDetails, localAuthorityCode);
+        String reference = Long.toString(caseDetails.getId());
+        String email = localAuthorityEmailLookupConfiguration.getLocalAuthority(localAuthorityCode).getEmail();
+        sendNotification(SDO_SUBMISSION_TEMPLATE, email, parameters, reference);
```

Comment:
Hmm... we are sending the same template here, but the template has got "Dear ((cafcass))" salutation maybe we need to handle that differently? I guess cafcass name is not correct if the notification is sent to the local authority.

Reply:
done - changed to title as discussed